tests/ast/README.md

```diff
@@ -12,7 +12,11 @@ N.B. No eager evaluation is permitted, as any intermediate batches will not be o
 df = session.table("test_table")
 df = df.filter("STR LIKE '%e%'")
 
-## EXPECTED OUTPUT
+## EXPECTED ENCODED AST
```

Comment:
could we maybe flip the order here? i.e. EXPECTED ENCODED AST as the last section?

I think it's much easier to read to have the base64 strings at the end.